packages/elements/src/components/ino-tooltip/ino-tooltip.tsx

```diff
@@ -182,14 +194,18 @@ export class Tooltip implements ComponentInterface {
   }
 
   render() {
-    const hostClasses = classNames(
-      `ino-tooltip--color-scheme-${this.colorScheme}`
-    );
-
     return (
-      <Host class={hostClasses}>
+      <Host>
         <div class="ino-tooltip__composer" role="tooltip">
-          <div class="ino-tooltip__inner">{this.label ? this.label : <slot />}</div>
+          <div class="ino-tooltip__inner">
+            {this.label ? 
+              this.label 
+              : 
+              <div>
+                {this.headerText && <header>{this.headerText}</header>}
```

Comment:
IMO the `header` should be independent of the existence of `label`.

Rougly like this:
```jsx
{header && <header> ... </header>}
{this.label ?? <slot />}
```